tests/test_neuropixels/test_open_ephys_rig_no_update.py

```diff
@@ -0,0 +1,63 @@
+"""Tests for the neuropixels open ephys rig ETL modification date updates."""
+
+import os
+import unittest
+from pathlib import Path
+
+from aind_data_schema.core.rig import Rig  # type: ignore
+from aind_metadata_mapper.neuropixels.open_ephys_rig import (  # type: ignore
+    OpenEphysRigEtl,
+)
+
+RESOURCES_DIR = (
+    Path(os.path.dirname(os.path.realpath(__file__)))
+    / ".."
+    / "resources"
+    / "neuropixels"
+)
+
+
+class TestOpenEphysRigEtlModificationDateUpdate(unittest.TestCase):
```

Comment:
rather than making a new module and class, you can put these unit tests in the test_open_ephys_rig module

Reply:
Addressed here: 
d3a041f